packages/components/src/TimeInput.tsx

```diff
@@ -1,9 +1,15 @@
-import React, { useEffect, useState } from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
 import Log from '@deephaven/log';
 import { TimeUtils } from '@deephaven/utils';
 import MaskedInput, {
   DEFAULT_GET_PREFERRED_REPLACEMENT_STRING,
-  SelectionSegment,
+  SelectionSegment as MaskedInputSelectionSegment,
 } from './MaskedInput';
 
```

Comment:
You don't need to rename this export, you can just do a separate export statement after, e.g.:
```suggestion
  SelectionSegment,
} from './MaskedInput';

export type { SelectionSegment } from './MaskedInput';

```